### PR TITLE
Add cookie and privacy policy page

### PR DESCRIPTION
**Pull request by damien-rembert**
Added a page for the cookie and privacy policies.
Slightly edited the text JC and Didier had written (eg adding a cookie policy).
If they don't mind having a little read, that would be much appreciated!
#64

### CODE REVIEW

**Comment by miurahr** [comment]
LGTM.

File: _i18n/en/policies.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  <p>The user or their project manager can set OmegaT to use online services. In that case, OmegaT shares information
+    with online services:</p>
+
+  <ul>
+    <li>Access to machine translation services</li>
+    <li>Access to team projects</li>
+  </ul>
+
+  <p>OmegaT automatically checks the availability of newer version on its development server but the user can easily opt
+    out in the Preferences settings. and does not transmit any user information.</p>
```
Comment (brandelune) [inline _i18n/en/policies.html:32]:
```suggestion
    out in the Preferences settings and does not transmit any user information.</p>
```

File: _i18n/en/policies.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  <p>If you find through your analysis that OmegaT communicates with an online service that you have not set, that would
+    be a
+    bug and we would ask you to report it.</p>
+
+  <p>The translatable data (ie. the files you are translating) are accessed to display the translatable contents to the
+    user.</p>
+
+  <p>The translated data is stored in a file called project_save.tmx located in the omegat/folder of any translation
```
Comment (brandelune) [inline _i18n/en/policies.html:41]:
```suggestion
  <p>The translated data is stored in a file called project_save.tmx located in the omegat/ folder of any translation
```

**Review by brandelune — CHANGES_REQUESTED** [review]

**Comment by damien-rembert** [comment]
Thanks for the edits! 
Changes applied.

File: _i18n/en/policies.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  <p>The user or their project manager can set OmegaT to use online services. In that case, OmegaT shares information
+    with online services:</p>
+
+  <ul>
+    <li>Access to machine translation services</li>
+    <li>Access to team projects</li>
+  </ul>
+
+  <p>OmegaT automatically checks the availability of newer version on its development server but the user can easily opt
```
Comment (brandelune) [inline _i18n/en/policies.html:31]:
```suggestion
  <p>OmegaT automatically checks the availability of newer versions on its development server, and the user can easily opt
```

File: _i18n/en/policies.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    out in the Preferences settings and does not transmit any user information.</p>
```
Comment (brandelune) [inline _i18n/en/policies.html:32]:
```suggestion
    out in the Preferences settings.</p>
```

Comment (brandelune) [reply _i18n/en/policies.html:32]: Apologies. I missed a glitch and a rewording. We should be good now. Thank you very much.

Comment (damien-rembert) [reply _i18n/en/policies.html]: Thank you! 
I'd rather take a bit longer and not leave typos and a public facing page :)

Comment (brandelune) [reply _i18n/en/policies.html]: Et joyeux Noël. On est déjà le 25 ici... :-)

Comment (damien-rembert) [reply _i18n/en/policies.html]: Joyeux Noël!

**Comment by brandelune** [comment]
Is there anything else to do on the PR?

**Review by brandelune — APPROVED** [review]

**Comment by brandelune** [comment]
@briacp Would you mind merging the PR?

**Pull request merged**